data-prepper-plugins/opensearch/security.md

```diff
@@ -42,13 +42,13 @@ Please check this [doc](https://docs.aws.amazon.com/elasticsearch-service/latest
 
 ### Fine-Grained Access Control (FGAC) in Amazon Elasticsearch Service
 
-The Elasticsearch sink creates an [Index State Management (ISM)](https://opendistro.github.io/for-elasticsearch-docs/docs/ism/) policy for Trace Analytics indices but Amazon Elasticsearch Service allows only the `master user` to create an ISM policy. So,
+The OpenSearch sink creates an [Index State Management (ISM)](https://opendistro.github.io/for-elasticsearch-docs/docs/ism/) policy for Trace Analytics indices but Amazon Elasticsearch Service allows only the `master user` to create an ISM policy. So,
  
  * If you use IAM for your master user in FGAC domain, configure the sink as below,
   
   ```
   sink:
-      elasticsearch:
+      opensearch:
         hosts: ["https://your-fgac-amazon-elasticssearch-service-endpoint"]
```

Comment:
Wondering if this should be updated too, even though there isn't an Amazon hosted OpenSearch service

Reply:
Tend to push back that change.